db/schema.rb

```diff
@@ -103,9 +103,9 @@
     t.string "unlock_token"
     t.datetime "locked_at"
     t.string "true_layer_secure_data_id"
+    t.boolean "employed"
```

Comment:
This change seems to creep in to migrations sometimes. I'm not entirely sure why but it shouldn't really be there. You can undo it by running:

```
rails db:rollback
git checkout origin/main db/schema.rb
rails db:schema:load
rails db:migrate
```
This should reset the schema and then reapply your migration but without the extra changes on lines 106 and 605.

Warning: `rails db:schema:load` will wipe your local db 😬